.travis.yml

```diff
@@ -8,6 +8,8 @@ addons:
     packages:
       - inkscape
       - kicad
+env:
+  -CACHED_BUILD=cashed
```

Comment:
You need a space after the dash (`-`) because YAML is weird. Check with http://www.yamllint.com/ if you are unsure.